Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -88,12 +89,14 @@ public class HTTPNetworkTransport {
               session: URLSession = .shared,
               sendOperationIdentifiers: Bool = false,
               useGETForQueries: Bool = false,
-              delegate: HTTPNetworkTransportDelegate? = nil) {
+              delegate: HTTPNetworkTransportDelegate? = nil,
+              requestCreator: RequestCreator? = nil) {
     self.url = url
     self.session = session
     self.sendOperationIdentifiers = sendOperationIdentifiers
     self.useGETForQueries = useGETForQueries
     self.delegate = delegate
+    self.requestCreator = requestCreator ?? ApolloRequestCreator()
```

Comment:
I think it would make more sense to provide `ApolloRequestCreator` as  the default value here - then the parameter doesn't have to be nullable